cc3d/core/Configuration/Configuration.py

```diff
@@ -0,0 +1,286 @@
+import os
```

Comment:
general question: do we want to store settings in the cc3d/core or are settings part of the player. so far settings were part of the player and I think this is where they belong. Also Twedit has separate settings. My preference would be to keep settings close to the code that they belong to because if we separate Player and the core code then we will end up with player settings being in different repository that actual player. But maybe I am missing something here

Reply:
OK I see the reason. Off screen rendering uses configuration. ugh... I guess leaving configuration in cc3d/core is one form of solution. If that's the case than core code is a required dependency of the Player. I think it could work . we may want to think about expanding Configuration package with some form of introspection that would allow command line only users to view and modify settings without using PLayer. A simple , user friendly API for accessing configuration